src/kernels/variables/debuggerVariables.node.ts

```diff
@@ -319,8 +319,8 @@ export class DebuggerVariables
             // Run our dataframe scripts only once per session because they're slow
             const key = this.debugService.activeDebugSession?.id;
             if (key && !this.importedDataFrameScriptsIntoKernel.has(key)) {
-                const scriptPath = path.join(EXTENSION_ROOT_DIR, DataFrameLoading.ScriptPath);
-                const contents = await this.fs.readLocalFile(scriptPath);
+                const scriptPath = DataFrameLoading.getScriptPath(this.context);
```

Comment:
YOu should be able to make this entire file 'debuggerVariables.ts'? Or did you want to wait for the next PR?

Reply:
I'm on it, I'm on it! Next push 🙂